persistence/patch-gcs.yml

```diff
@@ -0,0 +1,18 @@
+#-----------------------------------------------------------------------------------------------------------------
+# Example configuration for using a S3 bucket as spinnaker persistent store
+#-----------------------------------------------------------------------------------------------------------------
+apiVersion: spinnaker.armory.io/v1alpha2
+kind: SpinnakerService
+metadata:
+  name: spinnaker
+spec:
+  spinnakerConfig:
+    config:
+      persistentStorage:
+        persistentStoreType: gcs
+        gcs:
+          project: cloud-spinnaker              # The name of the GCP project that contains the bucket.
+          jsonPath: gcs-account.json            # The path to a JSON service account that has read/write permissions for the bucket.
```

Comment:
In practice this field would be an `encryptedFile` reference to be able to store the json file in a supported secret engine.
Since this repo uses k8s secret engine as an example, this field could be set to `encryptedFile:k8s!n:spin-secrets!k:gcs.json`, like in the example for GCS artifact: https://github.com/armory/spinnaker-kustomize-patches/blob/master/accounts/gcs/patch-gcs.yml#L16 .
In this way, people can drop the `gcs.json` file in `secrets/files` and this config would work out of the box when using the `deploy.sh` script.

Reply:
Added as suggested.